disnake/guild.py

```diff
@@ -4553,3 +4566,468 @@ async def create_automod_rule(
             reason=reason,
         )
         return AutoModRule(data=data, guild=self)
+
+
+PlaceholderID = NewType("PlaceholderID", int)
+
+
+class GuildBuilder:
+    """
+    A guild builder object, created by :func:`Client.guild_builder`.
```

Comment:
Generally, this shouldn't have an enter, but this hasn't been consistent lately, mostly with new classes (I'll probably create a PR to change all that)

Make sure to change it in other methods below 😄

Reply:
Okay never mind, just saw #722 and this can be changed in that PR

Reply:
you can mark this as resolved too